src/clients/mocks/MockSpokePoolClient.ts

```diff
@@ -108,7 +107,7 @@ export class MockSpokePoolClient extends SpokePoolClient {
       });
 
     // Update latestDepositIdQueried.
-    const idx = eventsToQuery.indexOf("V3FundsDeposited");
```

Comment:
This looks like one of those instances where we used the fact that deposit IDs are incrementing to store the latest deposit ID. I think this means that there will now be just garbage in `this.latestDepositIdQueried`, which I _think_ is alright, but flagging since we may just want to remove that variable entirely since it will contain nothing of substance.

Reply:
I can mark this resolved now that we dropped that part of SpokePoolClient update entirely in this commit: https://github.com/across-protocol/sdk/commit/d804fbe2e207047309a567823a84e27f7d76c032.